cloudsync/api.py

```diff
@@ -316,7 +318,9 @@ def parse_lecture_video_filename(filename):
           r'.*\.\w')  # Rest of filename including extension (required)
     matches = re.search(rx, filename)
     if not matches or len(matches.groups()) != 5:
-        log.exception('No matches found for filename %s with regex %s', filename, rx)
+        log.exception('No matches found for filename %s with regex %s',
+                      positional_args=(filename, rx),
+                      filename=filename, regex=rx)
```

Comment:
I think this can now be shortened to:
```python
log.exception('No matches found for filename %s with regex %s',
                      positional_args=(filename, rx))
```
because `filename` and `regex` aren't in the tags

Reply:
I'll make a commit to add `filename` to the tags, because I could see it being useful, but I'll remove `rx` from the arguments to the `exception()` call because I don't anticipate anyone searching on that.

Reply:
`filename` is in the tags [here](https://github.com/mitodl/odl-video-service/blob/6f941e1a3bf77e4efa5b261e7554a5e7e2ce5e32/odl_video/logging.py#L33) so I'll just get rid of `regex=rx` in the function args.

Reply:
Changed in 17e0a9e